scripts/publish/aws-s3.sh

```diff
@@ -30,21 +30,24 @@ function usage() {
   echo "    -f <file>"
   echo "    -b <s3 bucket>"
   echo "    -v <version>"
-  echo "    -p <product name>"
+  echo "    -n <product name>"
+  echo "    -p S3 key prefix"
```

Comment:
I think it'd be better to not change the meanings of existing flags.
You could add `-k` for **k**ey prefix? ;)